generate_postmortem.sh

```diff
@@ -136,6 +135,7 @@ for switch in $@; do
 
             IS_OVA=1
             NO_PROMPT=1
+            PULL_APPLIANCE_LOGS=1
```

Comment:
perhaps simpler to get rid of `PULL_APPLIANCE_LOGS` and just use `IS_OVA`

Reply:
Done